Sources/ToggleManager/ToggleManager.swift

```diff
@@ -6,6 +6,11 @@ import Foundation
 /// Thread-safe facade to interface with toggles.
 final public class ToggleManager: ObservableObject {
     
+    public enum ProviderValueError: Error {
```

Comment:
This is best placed in `Logging.swift` before the protocol definition.
Also best to not use the word provider at this level, I would suggest `ToggleError`.